File: TOI-Zero_68/A1/026/026.py
a = int(input())
b = int(input())
c = int(input())
odd = 0
even = 0
if a % 2 == 1:
    odd += 1
else:
    even += 1

if b % 2 == 1:
    odd += 1
else:
    even += 1

if c % 2 == 1:
    odd += 1
else:
    even += 1

print(f"even {even}")
print(f"odd {odd}")

File: TOI-Zero_68/A1/026/026.c
#include <stdio.h>

int main() {
    int a, b, c;
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);
    
    int odd = 0, even = 0;
    
    if (a % 2 == 1) {
        odd++;
    } else {
        even++;
    }
    
    if (b % 2 == 1) {
        odd++;
    } else {
        even++;
    }
    
    if (c % 2 == 1) {
        odd++;
    } else {
        even++;
    }
    
    printf("even %d\n", even);
    printf("odd %d\n", odd);
    
    return 0;
} 